udfs/community/corr_pvalue.sql

```diff
@@ -0,0 +1,18 @@
+-- corr_pvalue:
+-- Input:
+--     r: correlation value, n : number of samples
+-- Output: The p value of the correlation 
+CREATE OR REPLACE FUNCTION fn.corr_pvalue(r FLOAT64, n INT64 ) 
+RETURNS FLOAT64 
+LANGUAGE js AS """
+var abs_r = Math.abs(r)
```

Comment:
Looks like this statement "Math.abs(r)" is generating errors in the testing process(E   AssertionError: TypeError: "myproject.Math_test_local_test.abs" is not a function at UDF$1(FLOAT64, INT64))

; but this works fine in the console. Is this the right way to call the abs() function?